Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -482,6 +486,24 @@ static HRESULT _createPathReadyForFigure(CGPathRef previousPath,
     return S_OK;
 }
 
+// This function will return a normalized angle in radians between 0 and 2pi. This is to standardize
+// the calculations for arcs since 0, 2pi, 4pi, etc... are all visually the same angle.
+static CGFloat __normalizeAngle(CGFloat originalAngle) {
+    CGFloat returnAngle = fmod(originalAngle, 2 * M_PI);
+    if (returnAngle < .00001) {
+        returnAngle = 0;
+    }
+    if (returnAngle == 0) {
+        if (abs(originalAngle) > 0) {
```

Comment:
nit: this would be more clear as (originalAngle != 0) since you're taking the abs